.github/workflows/ci.yaml

```diff
@@ -37,3 +40,34 @@ jobs:
         with:
           name: quarto-vscode-${{ github.sha }}
           path: ${{ steps.package_extension.outputs.vsixPath }}
+  
+
+  publish-extension:
+    runs-on: ubuntu-latest
+    if: |
+      github.event_name == 'release' &&
+      github.repository_owner == 'quarto-dev'
+    needs:
+      - package-extension
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: latest
+      - run: yarn install --immutable --immutable-cache --check-cache
+      - uses: actions/download-artifact@v4
+        with:
+          name: quarto-vscode-${{ github.sha }}
+
+      - name: Publish to Open VSX Registry
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: ${{ secrets.OPEN_VSX_TOKEN }}
```

Comment:
This is the first of two tokens we will need added to the repo secrets to make this work.

Reply:
JJ added this a couple of days ago to this repo's secrets. ✅